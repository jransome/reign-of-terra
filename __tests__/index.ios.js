import 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <reign_of_terra />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
