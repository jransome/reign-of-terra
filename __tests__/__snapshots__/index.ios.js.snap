// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `<reign_of_terra />`;
